### DISCUSSION TITLE
PyMeshLab 2023.12 is out!

### DISCUSSION THREAD

**Discussion by alemuntoni** [cnr-isti-vclab/PyMeshLab#349] [General]
This versions brings some new filters, features and bugfixes:

- Based on [MeshLab 2023.12](https://github.com/cnr-isti-vclab/meshlab/discussions/1449);
- All the deprecated filter functions of version 2022.02 have been removed;
- The Mesh class allows to import and export additional data, like texture coordinates and transform matrix;
- Classes `Percentage` and `AbsoluteValue` have been renamed to `PercentageValue` and `PureValue`;
- Python 3.12 is now supported.

**Comment by xwqtju** [comment]
Does it support mac m1 pro?

**Reply by alemuntoni** [reply]
See #167.
Right now it doesn't. The idea was to compile it manually for a singular python version, but it doesn't work and I need time to figure out why.

**Reply by henrik-humlab** [reply]
Much appreciated if it would support m3 pro!

**Comment by NasserYassine** [comment]
@alemuntoni 
**How to use "save snapshot" in PyMeshLab !??**

**Reply by alemuntoni** [reply]
Just don't: #164
